src/core/geometry_func_unittest.cpp

```diff
@@ -0,0 +1,33 @@
+/* $Id$ */
+
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file geometry_func_unittest.cpp Geometry functions unit tests. */
+
+#include "../stdafx.h"
+#include "geometry_func.hpp"
+#include "math_func.hpp"
+#include "gtest/gtest.h"
```

Comment:
```suggestion
#include <gtest/gtest.h>
```
I think it should be treated as a system include.